webapp/pages/project_page/builds_tab.js

```diff
@@ -9,7 +9,7 @@ import { Menu1 } from 'es6!display/menus';
 import { SingleBuildStatus } from 'es6!display/changes/builds';
 import { TimeText, display_duration } from 'es6!display/time';
 import { get_build_cause } from 'es6!display/changes/build_text';
-import { get_runnable_condition, is_waiting } from 'es6!display/changes/build_conditions';
+import { is_waiting, get_runnable_condition } from 'es6!display/changes/build_conditions';
```

Comment:
Also shuffled out of order here.

How does the import script know when to break imports over multiple lines?